config/sidekiq_data.yml

```diff
@@ -1,5 +1,8 @@
 :concurrency: <%= ENV.fetch("REDIS_QUEUE_MAX_THREADS", 5).to_i - 2 %>
 :queues:
+  - create_debits_for_campaign_and_date
+  - create_debit_for_campaign_and_date
+  - recalculate_organizataion_balances
   - ensure_daily_summaries
   - daily_summaries
   - daily_summary
```

Comment:
Does anything need to be removed from this list?

Reply:
Nope all are required and very explicit to better control the order of operations and priority.